src/localization/gps_linearization.py

```diff
@@ -3,10 +3,11 @@
 from util.SE3 import SE3
 from mrover.msg import ImuAndMag
 from sensor_msgs.msg import NavSatFix
+from geometry_msgs.msg import PoseWithCovarianceStamped, PoseWithCovariance, Pose, Point, Quaternion
+from std_msgs.msg import Header
 import tf2_ros
 import numpy as np
 from pymap3d.enu import geodetic2enu
-from tf.transformations import quaternion_about_axis, quaternion_multiply
 
 
 class GPSLinearization:
```

Comment:
This is not something I want you to change in this PR but I think the structure of GPS Linearization could usee work and we could refactor this later in a way that creates a better interface. Fundamentally I feel like the GPS linearization stuff should be more like "library code" and recieving data in callbacks shouldn't be part of this. Instead, the interface boundary would be the info needed for one pass at a linearization, and some other wrapper has the statefulness and callbacks. Lets open an issue for a refactor and discuss later.